test/detect/index.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
-let bluebird = require('bluebird');
-let fs       = bluebird.promisifyAll(require('fs'));
+let fs       = require('fs').promises
```

Comment:
```suggestion
const fs = require('fs').promises
```

Reply:
Sure if we're doing that I'll change them all.  I was doing the minimum work, not rewriting. :-)  But I can.